extensions/binding/org.eclipse.smarthome.binding.homematic/README.md

```diff
@@ -189,6 +189,7 @@ Bridge homematic:bridge:ccu [ gatewayAddress="..." ]
 
 The first parameter after Thing is the device type, the second the serial number.
 If you are using Homegear, you have to add the prefix ```HG-``` for each type.
+The prefix ```HG-```is needed for Things and Items but not Types.
```

Comment:
From the code it looks to me as if the prefix is only used in ThingTypeUids (neither for Things nor for Items). From that perspective the original line was correct.